### DISCUSSION TITLE
Question while running

### DISCUSSION THREAD

**Discussion by anishshriram** [mlcommons/GaNDLF#323] [Q&A] [answered]
**GaNDLF Version**
0.0.14-dev

**Desktop**
macOS Monterey
Version 12.1

**How did you install GaNDLF**
```
git clone https://github.com/CBICA/GaNDLF.git
cd GaNDLF
conda create -n venv_gandlf python=3.7 -y
conda activate venv_gandlf
conda install -c conda-forge mamba -y
mamba install -c pytorch pytorch torchvision -y
pip install -e .
```

**Dataset description**
The dataset is titled RSNA_ASNR_MICCAI_BraTS2021_TrainingData_16July2021, referring to the 2021 BraTS training data.

Description from website:
Routine clinically-acquired multi-parametric MRI (mpMRI) scans of glioma, with pathologically confirmed diagnosis and available MGMT promoter methylation status (for the glioblastoma cases with such associated data). All BraTS mpMRI scans are available as NIfTI files (.nii.gz) for Segmentation, which is the use in this issue. These mpMRI scans describe a) native (T1) and b) post-contrast T1-weighted (T1Gd), c) T2-weighted (T2), and d) T2 Fluid Attenuated Inversion Recovery (T2-FLAIR) volumes.

**Describe your question/problem**
I am trying to use gandlf_run.

I then executed it (with specific paths) as so:
```
python gandlf_run \
-c /cbica/home/shrirama/comp_space/project/meningioma/GaNDLF/samples/config_segmentation_brats.yaml \
-i /cbica/home/shrirama/comp_space/project/meningioma/data_processed.csv \
-m /cbica/home/shrirama/comp_space/project/meningioma/models \
-t True \
-d cuda \
-r True
```

I have been coming across two following errors, both of which I have no idea how to solve.

The first: 
```
RuntimeError: Exception thrown in SimpleITK ImageFileReader_ReadImageInformation: /tmp/SimpleITK/Code/IO/src/sitkImageReaderBase.cxx:105:
sitk::ERROR: Unable to determine ImageIO reader for "/cbica/home/shrirama/comp_space/project/meningioma/BraTS2021_01165/BraTS2021_01165_3.nii.gz"
```

I assume this has something to do with the config file. I am using an edited version of the sample given --> ['config_segmentation_brats.yaml']:

final layer:
sigmoid to softmax

class list:
 '[1||2||4,1||4,4]' to '[1,2,4]'

patch size:
[128,128,128] to [64,64,64]

loss_function:
dcce to dc

optimizer
adam to sgd

I also commented out the line for parallel training

I would appreciate some help on this thank you so much.

**Comment by sarthakpati** [comment, marked as answer]
This is because the file is either not present or corrupted:

```bash
(mamba) patis@/cbica/home/patis>$ file /cbica/home/shrirama/comp_space/project/meningioma/BraTS2021_01165/BraTS2021_01165_3.nii.gz
/cbica/home/shrirama/comp_space/project/meningioma/BraTS2021_01165/BraTS2021_01165_3.nii.gz: empty
```

**Reply by anishshriram** [reply]
Alright. I will try to upload the data again.
